### PR TITLE
Sticky QuickPurchase form

### PR DESCRIPTION
**Pull request by gustaveen**
## Describe your changes
Add Sticky **See your price** button to ProductPageV2 for mobile

https://github.com/user-attachments/assets/d5efd6b0-7b1b-40d8-918d-5d51a929ea7a

Screenshot 2024-09-19 at 10 28 57

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by gustaveen** [comment]
* **#4751** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4751?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @gustaveen and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/store/src/components/QuickPurchaseV2/QuickPurchaseForm/QuickPurchaseForm.css.ts [hunk 1]
```diff
@@ -36,3 +36,18 @@ export const productSingleOption = style([
...
+  justifyContent: 'center',
+  paddingInline: tokens.space.md,
+  '@media': {
+    [minWidth.lg]: { display: 'none' },
+  },
+})
+
+export const stickyButton = style({
+  '@media': {
+    [minWidth.xs]: { maxWidth: '18.75rem' },
```
Comment (gustaveen) [inline apps/store/src/components/QuickPurchaseV2/QuickPurchaseForm/QuickPurchaseForm.css.ts:51]: Limit width on larger devices where we don't want 100% width

**Review by alebedev — APPROVED** [review]
Nice!

**Review by alebedev — COMMENTED** [review]
Do we want to tweak appear animation while we're touching it?

**Review by guilhermespopolin — APPROVED** [review]
Looking good :)

**Comment by gustaveen** [comment]
> Do we want to tweak appear animation while we're touching it?

Good idea, let's check with nils if we want to adjust it @alebedev

**Pull request merged**